agent/token/store.go

```diff
@@ -287,3 +290,46 @@ func (t *Store) IsAgentMasterToken(token string) bool {
 
 	return (token != "") && (subtle.ConstantTimeCompare([]byte(token), []byte(t.agentMasterToken)) == 1)
 }
+
+type Logger interface {
+	Warn(msg string, args ...interface{})
+}
+
+type Config struct {
+	EnablePersistence   bool
+	DataDir             string
+	ACLDefaultToken     string
```

Comment:
What should this `ACLDefaultToken` be called? We seem to call it a user token sometimes, and a default token other times. I'm not sure which is correct. I guess the user token acts as a defualt, so maybe `ACLUserToken` is more correct?

I'm also wondering if the `ACL` prefix is redundant.